src/library/scala/collection/immutable/LazyList.scala

```diff
@@ -377,14 +428,23 @@ sealed private[immutable] trait LazyListOps[+A, +CC[+X] <: LinearSeq[X] with Laz
     sb
```

Comment:
Does `addString` need to call `force` anymore, since it's no longer inherited from by `Stream`? Or to put it differently, can `toString` call `addString` instead of `addStringNoForce`?

Reply:
`addString` forces the `LazyList` in order to be consistent with what `mkString` does